src/arviz_plots/plots/energyplot.py

```diff
@@ -0,0 +1,138 @@
+    in practice are challenging for sampling.
+
+    This plot is in the style of the one used in [1]_.
+
+    Parameters
+    ----------
+    dt : DataTree
+        ``sample_stats`` group with an ``energy`` variable is mandatory.
+    bfmi : bool
+        Whether to the plot the value of the estimated Bayesian fraction of missing
+        information. Defaults to False. Not implemented yet.
+    kind : {"kde", "hist", "dot", "ecdf"}, optional
+        How to represent the marginal density.
+        Defaults to ``rcParams["plot.density_kind"]``
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh", "plotly"}, optional
+    labeller : labeller, optional
+    aes_map : mapping of {str : sequence of str}, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `plot_kwargs`.
+
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * One of "kde", "ecdf", "dot" or "hist", matching the `kind` argument.
+
+          * "kde" -> passed to :func:`~arviz_plots.visuals.line_xy`
+          * "ecdf" -> passed to :func:`~arviz_plots.visuals.ecdf_line`
+          * "hist" -> passed to :func: `~arviz_plots.visuals.hist`
+
+        * title -> passed to :func:`~arviz_plots.visuals.labelled_title`
+        * remove_axis -> not passed anywhere, can only be ``False`` to skip calling this function
+
+    stats_kwargs : mapping, optional
+        Valid keys are:
+        * density -> passed to kde, ecdf, ...
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection.wrap`
+
+    Returns
+    -------
+    PlotCollection
+
+    Examples
+    --------
+    Plot a default energy plot
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_energy, style
+        >>> style.use("arviz-clean")
+        >>> from arviz_base import load_arviz_data
+        >>> schools = load_arviz_data('centered_eight')
+        >>> plot_energy(schools)
+
+
+    .. minigallery:: plot_energy
+
+    """
+    if kind is None:
+        kind = rcParams["plot.density_kind"]
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    else:
+        plot_kwargs = plot_kwargs.copy()
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    new_ds = _get_energy_ds(dt)
+    pc_kwargs.setdefault("cols", ["__variable__"])
+    pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+    pc_kwargs["aes"].setdefault("color", ["energy"])
+    plot_kwargs.setdefault("credible_interval", False)
+    plot_kwargs.setdefault("point_estimate", False)
+    plot_kwargs.setdefault("point_estimate_text", False)
+    plot_kwargs.setdefault("title", False)
+
+    plot_collection = plot_dist(
+        new_ds,
+        var_names=None,
+        filter_vars=None,
+        group=None,
+        coords=None,
+        sample_dims=None,
```

Comment:
I don't think we want this. Sample dims should either be an input and passed downstream to plot_dist or if having flattened to `sample` is not to be supported here then we should hardcode it and check both `chain` and `draw` are present.